app/javascript/components/table/TableHead.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div class="table__head">
+    <div v-for="(heading, index) in headings"
```

Comment:
Would span be more appropriate here? I usually think of div as full width or not positioned within the natural flow of the page and then span as inline. Not sure if I'm right there.